src/tools/miri/README.md

```diff
@@ -212,7 +212,6 @@ degree documented below):
 - All Rust [Tier 1 targets](https://doc.rust-lang.org/rustc/platform-support.html) are supported by
   Miri. They are all checked on Miri's CI, and some (at least one per OS) are even checked on every
   Rust PR, so the shipped Miri should always work on these targets.
-- `aarch64-apple-darwin` is supported.
```

Comment:
[This comment here](https://github.com/rust-lang/rust/blob/2d33d7bc6236b2c9add0aec9befa1459ef160583/src/tools/miri/ci/ci.sh#L140) also needs updating now.